web/profiles/joinup/joinup.behat.inc

```diff
@@ -388,7 +389,7 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
    * This allows us to:
    * - use human readable labels like 'short title' and 'collection' in test
    *   scenarios, and map them to the actual field names like
-   *   'field_document_short_title' and 'og_audience'.
+   *   'field_document_short_title' and 'OgGroupAudienceHelper::DEFAULT_FIELD'.
```

Comment:
This does not seem good to me. 
Replacing the value also in documentation, seems kind of confusing. This is explanation, not coding.